hl7/4.2/modules/ROOT/edits/hl7-connector-config-topics.adoc

```diff
@@ -0,0 +1,269 @@
+= HL7 EDI - Additional Configuration Information - Mule 4
+
+Use these options to:
+
+* Customize your schema Definitions
+
+* Define a mapping from the HL7 event and message types to message structures
+
+* Customize global elements
+
+== Copy and Edit Schema Definitions
+
+HL7 schema definitions are located in the `hl7-schemas-4.0.0.jar` file,
+embedded inside HL7 EDI Connector update site and can also
+be found in the standard MuleSoft enterprise Maven repositories
+(under group ID `com.mulesoft.connectors`). You can copy a message structure schema
+from this JAR file and edit it to match your specific needs.
+
+To make changes at the segment level, copy the message structure
+schema. To change segment definitions, copy the `basedefs.esl` schema of the same version, as that file contains the segment, composite, and element definitions.
```

Comment:
Are Lines 16-20 supposed to be three examples to match the three places to find the schema definitions?
If so, I would place each example with its location.
This might be more clear if we used a bullet list of the three locations, followed by their respective descriptions.

Reply:
STET